pom.xml

```diff
@@ -1823,7 +1823,9 @@
       <id>flink1.15</id>
       <properties>
         <flink.version>${flink1.15.version}</flink.version>
+        <flink.runtime.artifactId>flink-runtime</flink.runtime.artifactId>
         <flink.table.runtime.artifactId>flink-table-runtime</flink.table.runtime.artifactId>
+        <flink.table.planner.artifactId>flink-table-planner_2.12</flink.table.planner.artifactId>
```

Comment:
The default active profile is flink1.14, and it already specifies the jar artifactIds, so what's the problem here

Reply:
Fix the default flink version line 131 ~ line 133:

```xml
    <flink.version>${flink1.14.version}</flink.version>
    <hudi.flink.module>hudi-flink1.14.x</hudi.flink.module>
    <flink.bundle.version>1.14</flink.bundle.version>
```
should also work ?

Reply:
If not changing the artofact ids, then i think changing the default flink profile from flink1.13 to flink1.15 should work. I guess that's what #6445 is doing.